src/oauth/oauth.service.js

```diff
@@ -51,8 +51,8 @@ class OAuthAdapter{
         var deferred = this.$q.defer();
         options.path = '/api' + options.path;
         if (options.data) {
-            options.body = options.data;
-            options.data = undefined;
+            options.content = options.data;
```

Comment:
here you are change options.body by options.data.

Could you explain to me this change a bit more ?

I have not being so far in the use of oauth and there is no tests at all here.

Could you show me for example a doc or part of the underlying API which is in the sources ?

I don't want to block sorry for the delay to answer I promise no more delay like this one